examples/shop-page/Components/ShopPage.lua

```diff
@@ -0,0 +1,73 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Components = script.Parent
+
+local Roact = require(ReplicatedStorage.Roact)
+local ProductItemList = require(Components:WaitForChild("ProductItemList"))
+
+local ShopPage = Roact.Component:extend("ShopPage")
+
+ShopPage.defaultProps = {
+	padding = 0,
+}
+
+function ShopPage:init()
+	self:setState({
+		cellSize = 100,
+		canvasHeight = 100,
```

Comment:
Since we don't know the actual cellSize or canvasHeight until the first call to onAbsoluteSizeChanged, should we default these to zero and not render anything in that case?

Reply:
Sounds reasonable to me